vrpn_Spaceball.C

```diff
@@ -103,6 +103,8 @@ vrpn_Spaceball::vrpn_Spaceball (const char * name, vrpn_Connection * c,
 	vrpn_Button::num_buttons = _numbuttons;
 	vrpn_Analog::num_channel = _numchannels;
 
+        vrpn_gettimeofday(&timestamp, NULL);	// Set watchdog now
+
```

Comment:
Extra space (this is an easy one to see on my phone using the web interface which is why I'm typing so many characters instead of fixing it myself)